src/Servers/HttpSys/HttpSysServer.slnf

```diff
@@ -4,9 +4,11 @@
     "projects": [
       "src\\DefaultBuilder\\src\\Microsoft.AspNetCore.csproj",
       "src\\Extensions\\Features\\src\\Microsoft.Extensions.Features.csproj",
+      "src\\FileProviders\\Embedded\\src\\Microsoft.Extensions.FileProviders.Embedded.csproj",
```

Comment:
I needed these to be able to build/run the tests so I included but I can remove if this isn't desired.

Reply:
Fine to keep, these files often get out of sync and you only know when using a clean repo that hasn't built the dependent projects.